grid/pdf-export-server/Client/Services/PdfExportService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using Telerik.DataSource;
+
+namespace ServerPdfExport.Client.Services
+{
+    public class PdfExportService
+    {
+        [Inject]
+        private HttpClient Http { get; set; }
+
+        [Inject]
+        private IJSRuntime JS { get; set; }
+
+        public PdfExportService(HttpClient client, IJSRuntime _js)
```

Comment:
No need for the underscore before `_js`.. Can also be renamed to `JsRuntimeInstance` to be more descriptive and the injected variable to be `jsRuntime`.